src/services/rollouts.js

```diff
@@ -540,4 +543,216 @@ module.exports = class RolloutsHelper {
 			return error
 		}
 	}
+	/**
+	 * rollout publish
+	 * @method
+	 * @name publish
+	 * @param {Integer} rolloutId - rollout id
+	 * @returns {JSON} - rollout publish response.
+	 */
+
+	static async publish(rolloutId, orgId, loggedInUserId) {
+		try {
+			// fetch rollout details
+			const rolloutDetails = await this.details(rolloutId, orgId, loggedInUserId)
+
+			// check if rollout is present or not
+			if (rolloutDetails?.statusCode != 200 || rolloutDetails?.result == undefined) return rolloutDetails
```

Comment:
@adithyadinesh0412 rolloutDetails.statusCode !== httpStatusCode.ok and we dont need to check the result key , if its 200 anyway we are returning data right ?